src/lib/zork/zork.ts

```diff
@@ -0,0 +1,364 @@
+/// <reference path='../../../third_party/wrtc/wrtc.d.ts' />
+
+/*
+ * Pure Node implementation of zork (no freedomjs).
+ * Rough lifecycle is to process single word commands such as "ping" until
+ * "get" or "give" is received, at which point a p2p proxy session is
+ * established and further input is treated as signaling channel messages.
+ */
+
+import * as constants from '../../generic_core/constants';
+import * as net from 'net';
+import * as node_server from '../socks/node/server';
+import * as node_socket from '../socks/node/socket';
+import * as socks_session from '../socks/session';
+import * as wrtc from 'wrtc';
+
+const RTC_PEER_CONFIG = {
+  iceServers: [
+    {url: 'stun:stun.l.google.com:19302'},
+    {url: 'stun:stun1.l.google.com:19302'},
+    {url: 'stun:stun.services.mozilla.com'}
+  ]
+};
+
+// The delimiter for Zork messages is just \n, but we use \r?\n for
+// MSG_DELIM_RE so that \r\n sent by e.g. a telnet client will also be treated
+// as a delimiter. https://en.wikipedia.org/wiki/Robustness_principle
+// "Be conservative in what you send, be liberal in what you accept."
+const MSG_DELIM_RE = /\r?\n/;
+
+const SOCKS_HOST = '0.0.0.0';
+const SOCKS_PORT = 9999;
+
+const ZORK_SERVER_PORT_START = 9000;
+const ZORK_SERVER_PORT_INCREMENT = 10;
+const ZORK_SERVER_BIND_MAXTRIES = 3;
+let zorkServerPort = ZORK_SERVER_PORT_START;
+let zorkServerBindNtries = 0;
+let nconnectionsMade = 0;
```

Comment:
Done in b0d1a1d. Also prompted me to find https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers which also prescribes this style. (Though unfortunately that guide doesn't prescribe a single style for count variables. e.g. It sounds like it would have no problem with `connectionsCount` either.)